data-processing/tests/test_parse_equation.py

```diff
@@ -86,6 +86,22 @@ def test_get_symbol_children():
     assert t in t_sub_i.children
     assert i in t_sub_i.children
 
+def test_get_symbol_sub_super_with_numeric_child():
+    with open(
+        get_test_path(os.path.join("mathml", "x_sub_four_sub_three.xml"))
+    ) as mathml_file:
+        mathml = mathml_file.read()
+        symbols = get_symbols(mathml)
+
+    assert len(symbols) == 3
+    x_sub_four_sub_three = list(
+        filter(lambda s: "msub" in s.mathml and "x" in s.mathml, symbols)
+    )[0]
+    x = list(filter(lambda s: s.mathml == "<mi>x</mi>", symbols))[0]
+
+    assert len(x_sub_four_sub_three.children) == 2
+    assert x in x_sub_four_sub_three.children
```

Comment:
Can you add an assertion to check here that the nodes `4` and `3` are among in `x_sub_four_sub_three.characters`?